third_party/tsl/third_party/py/ml_dtypes/e8m0.patch

```diff
@@ -0,0 +1,99 @@
+diff --git a/include/float8.h b/include/float8.h
+index 51d91fd..c53eb45 100644
+--- a/include/float8.h
++++ b/include/float8.h
```

Comment:
Do you plan on submitting this patch to ml_dtypes? It might be easier to submit a PR to ml_dtypes before merging this PR.

Reply:
jax/ml-dtypes already has these changes: https://github.com/jax-ml/ml_dtypes/commit/47095b1d2d0c2aa382cf926e8ceaa2a13f0904c5
Once that change propagates to the XLA repo, this patch may be removed.